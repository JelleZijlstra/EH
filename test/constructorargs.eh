#!/usr/bin/ehi
# Constructors can now take arguments
class Foo {
  initialize = func: a, b {
    $echo a
    $echo b
  }
}
o1 = Foo.new: "a", "b"
o2 = Foo.new:
