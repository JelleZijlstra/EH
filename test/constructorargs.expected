a
b
Incorrect argument count for function: expected 2, got 1: EH runtime error
